caikit/runtime/train.py

```diff
@@ -130,15 +133,29 @@ def main() -> int:
     parser.add_argument(
         "--termination-log-file",
         "-f",
-        action="store_true",
-        default="/dev/termination-log",
         help="Location of where to write a termination error message",
     )
+    parser.add_argument(
```

Comment:
I think we can do a little better here by doing the `os.environ.get` before `parser.add_argument` and then using the value as the `default` in `parser.add_argument` above. This will allow the user to specify a value on the command line that would take precedence over the env (standard default -> env -> cli precedence)

Reply:
Ah yeah makes sense.

Reply:
So these lines effectively end up doing that (allowing the parser values to override this variable indicating the termination log variables). It's not as elegant as I'd like, but I needed to store the values in a variable that wasn't pulled directly from the parser - so in the cases where the parser threw errors, I'd still be able to access the termination log values.

Reply:
These lines: https://github.com/caikit/caikit/blob/c7e00e9742fed5b4605713e289cffb088b3f6822/caikit/runtime/train.py#L153-L157

Reply:
Ah, makes sense!